src/setup/rgw.py

```diff
@@ -504,8 +504,8 @@ def _logrotate_generic(conf):
         # '/etc/logrotate.d/ceph' -> '/etc/logrotate.d/radosgw'
         # currently component_name is 'rgw', file='/etc/logrotate.d/radosgw'
         old_file = os.path.join(LOGROTATE_DIR, 'ceph')
-        new_file = os.path.join(LOGROTATE_DIR, 'radosgw')
-        os.rename(old_file, new_file)
+        if os.path.exists(old_file):
+            os.remove(old_file)
         try:
```

Comment:
if old file does not exists and we want to copy our logrotate file,  then at which step we are copying this template file to /etc/logrotate.d directory ? 
is is part of rpm installation or will it be done as config phase

Reply:
@sachitanands Please check the try block. #L509
There we are copying our template file to /etc/logrotate.d directory.

Reply:
> is is part of rpm installation or will it be done as config phase

It will done in prepare phase